src/main/java/com/github/simonpercic/responseecho/GoogleApiTokenProvider.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.simonpercic.responseecho
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
+import org.springframework.core.io.ClassPathResource
+import java.util.*
+
+const val JSON_FILE_NAME = "prettylog_546b4_firebase_adminsdk_g2afy_917a2ad7d2.json"
+
+object GoogleApiTokenProvider {
+    fun getToken(): Pair<String?, Long> {
```

Comment:
Create a small data class, instead of using `Pair`. This will enable easier readability and understanding of what the fields are representing.